app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -29,19 +38,69 @@ export default function NewNoteForm({ onClose, refetch }) {
     >
       {({ isSubmitting }) => (
         <Form>
-          <Input label="Title" name="title" className="mb-6" />
-          <Textarea label="Description" name="description" rows={8} />
+          <Input label="Note Title" name="title" className="mb-6" />
+          <Select
+            className="mb-6"
+            label="Tags"
+            defaultValue={{ value: "internal", label: "Internal" }}
+            placeholder="Select a tag"
+            name="tags"
+            options={[
+              { value: "internal", label: "Internal", color: "blue" },
+              {
+                value: "agile_workflow",
+                label: "Agile Workflow",
+                color: "green",
+              },
+              { value: "bug", label: "Bug", color: "red" },
+            ]}
+          />
+          <Textarea
+            label="Note Description"
+            name="description"
+            rows={8}
+            className="mb-6"
+          />
+          <Select
+            className="mb-6"
+            label="Assigned Contact"
+            defaultValue={{ value: "internal", label: "internal" }}
+            placeholder="Select a contact"
+            name="contact"
+            options={[
+              { value: "tom hunk", label: "Tom hunk" },
+              { value: "amrut sabale", label: "Amrut Sabale" },
+              { value: "john smith", label: "John smith" },
+            ]}
+          />
+          <div className="mb-80">
+            <div className="flex justify-between mb-6">
+              <Label>Add Due Date to Note</Label>
+              <Switch
+                checked={isDueDateRequired}
+                onChange={e => setIsDueDateRequired(e.target.checked)}
+              />
+            </div>
+            {isDueDateRequired && (
+              <DateInput
+                label="Due date"
+                format="DD/MM/YYYY"
+                name="dueDate"
+                value={dueDate}
+                onChange={setDueDate}
+              />
+            )}
+          </div>
```

Comment:
Rather than using `mb-6` for each block individually, please use `space-y-6`.

Please refer [space between](https://tailwindcss.com/docs/space#class-reference) from tailwindCSS for more details.